plugin/trino-sqlserver/pom.xml

```
@@ -11,7 +11,7 @@
 
     <artifactId>trino-sqlserver</artifactId>
     <packaging>trino-plugin</packaging>
-    <description>Trino - SQL Server Connector</description>
+    <description>Trino - SQLServer connector</description>
```

```suggestion
    <description>Trino - SQL Server connector</description>
```

See https://trino.io/docs/current/connector/sqlserver.html